app/controllers/serials_controller.rb

```diff
@@ -1,6 +1,8 @@
 class SerialsController < ApplicationController
   def index
     @serials = current_site.serials.published.order(id: :desc).page(params[:page])
```

Comment:
OK. Give it an appropriate name as `scope`? It's the problem that we don't understand that from `order(id: :desc)`.